CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+### 0.5.0 - Dec 07, 2020
+
+* Renamed to [hiredis-cluster](https://github.com/Nordix/hiredis-cluster)
+* The C library `hiredis` is an external dependency rather than a builtin part
+  of the cluster client, meaning that `hiredis` v1.0.0 or later can be used.
+* Support for SSL/TLS introduced in Redis 6
+* Support for IPv6
```

Comment:
Mention Support authentication using AUTH?

Reply:
Ah, great, will do

Reply:
Handle variable number of keys in EXISTS also maybe? Or group EXISTS, EVAL and other improved commands in one bullet point.

Reply:
Added text for EXISTS, eval became only test-improvements. But there are changes that can be worked on for next release.